pkg/perf/networkhealth/networkhealth.go

```diff
@@ -45,6 +47,10 @@ func (t *NetworkHealthTask) Jitter() uint32 {
 }
 
 func (t *NetworkHealthTask) Run(ctx context.Context) (interface{}, error) {
+	if err := app.DeleteFlag(app.NotReachable); err != nil {
+		log.Error().Err(err).Msg("failed to unset not reachable flag")
+	}
+
```

Comment:
I am wondering if it's a good idea to delete the flag before running the test. What do you think?

Reply:
modified to delete the notreachable flag after the run only if there are no errors